packages/dev-config/.prettierrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  printWidth: 160,
+  tabWidth: 2,
+  useTabs: false,
+  semi: true,
+  quoteProps: "as-needed",
+  jsxSingleQuote: false,
+  bracketSameLine: false,
+  trailingComma: "none",
+  requirePragma: false,
+  insertPragma: false,
+  singleQuote: false,
+  arrowParens: "avoid",
+  endOfLine: "crlf",
+  htmlWhitespaceSensitivity: "strict",
+  plugins: ["prettier-plugin-tailwindcss"]
```

Comment:
Are the [overrides for tailwindConfig](https://github.com/akash-network/cloudmos/pull/218/commits/1937dcc56c7408b27d03a674d1b8bffa61b3a713#diff-663ade211b3a1552162de21c4031fcd16be99407aae5ceecbb491a2efc43d5d2) no longer needed here?

Reply:
They are in the root .prettierrc.js as package one is more general